server/src/main/java/org/spine3/server/aggregate/Aggregate.java

```diff
@@ -57,7 +57,7 @@
  *
  * <h2>Creating an aggregate class</h2>
  *
- * <p>In order to create a new aggregate class you need to:
+ * <p>In order to createCommand a new aggregate class you need to:
```

Comment:
Is this an automatic replacement? The text does not make any sense now. It's everywhere in the docs now. 

Please fix.